src/event.js

```diff
@@ -347,11 +347,11 @@ jQuery.event = {
 			delegateCount = handlers.delegateCount,
 			cur = event.target;
 
-		// Support (at least): Chrome, IE9
+		// Support (at least): IE 9
```

Comment:
OK, I dropped it. However, I think there's some value in stating explicitly that a particular scenario wasn't tested in at least the few major desktop browsers we support. Otherwise it's easy to remove the code when support for a particular browser is dropped only to be surprised later.

OTH, 98% of such code should have unit tests so we'd catch that immediately...